docs/DeveloperGuide.md

```diff
@@ -1,29 +1,204 @@
+2. delete_staff 
+
+    It first finds the corresponding `Staff` object to be deleted using its name in the list in `StaffManager`. 
+    If it is found, it will delete the correct corresponding `Staff`. 
+
+3. view_staff
+
+    It prints all of `Staff` objects in `StaffManager`'s staffs. 
+
+### Dish Feature (Zheng Rong)
+
+The Dish feature consists of three functions:
+
+##### Adding dish to list:
+
+![](uml/images/AddDishCommandSequenceDiagram-Add_Dish_Sequence_Diagram.png)
+
+- When the ```AddDishCommand()``` constructor is called, it stores the dish name, price and the list of ingredients in an entity called Dish.
+- When the ```execute()``` command in ```AddDishCommand``` is called, it calls the ```addDishCommand()``` in ```DishManager``` class that adds the Dish into an arraylist of Dishes.
+- It then prints out a message to the console that informs the user that a dish has been added.
+
+##### Deleting dish on the list:
+- When the ```DeleteDishCommand()``` constructor is called, it stores the index of the Dish to be deleted from the arraylist of Dishes.
+- When the ```execute()``` command in ```DeleteDishCommand``` is called, it calls the ```DeleteDishCommand()``` in ```DishManager``` class that deletes the Dish at the specified index in the arraylist of Dishes
+- It then prints out a message to the console that informs the user that a dish has been added.
+
+##### Viewing the list of dishes:
+- When the ```execute()``` command in ```ViewDishCommand``` is called, it calls the ```ViewDishCommand()``` in ```DishManager``` class that returns the formatted string of all the dishes in the arraylist.
+- It then prints out the formatted string to the console.
 
 ## Product scope
 ### Target user profile
-
-{Describe the target user profile}
+Restaurant Managers  
+1. The purpose of the product is to help restaurant managers to manage their restaurant in a more convenient manner.
+2. We aim to create an all in application where the restaurant manager can keep track of what is needed to run a restaurant daily.
 
 ### Value proposition
+There are too many things to keep track of in a restaurant. We want to create an application that will allow the restaurant manager to have a place to quickly refer to, and keep track of the things that allow the restaurant to function on a daily basis.
+
 
-{Describe the value proposition: what problem does it solve?}
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...                                                 | I want to ...                                            | So that I can ...                                                                         |
```

Comment:
I really liked the user stories and I think they all fit the purpose of your product. However, it would be better to make it more consistent. For the `so that i can...` section, some of the individual cells start with `so that i can...` or `i can...`. I would recommend removing these to make it more consistent throughout and to be in-line with table headings